music21/clef.py

```diff
@@ -365,13 +365,26 @@ class TabClef(PitchClef):
     >>> a = clef.TabClef()
     >>> a.sign
     'TAB'
+    
+    As tab stems are assumed to be mostly downwards (at least for guitar tabs),
+    the method `getStemDirectionForPitches` is overriden from `Clef` to
```

Comment:
overridden has two "d"s (gets me all the time).

Also, this documentation is already on `getStemDirectionForPitches`, so I don't think you need to refer to the name of the function, but mentioning the polymorphism is good! Something simple like "Overridden to simply return 'down' for guitar tabs." would do, IMO.